include/oneapi/dpl/internal/serial_algorithm_impl.h

```diff
@@ -17,7 +17,9 @@
 #ifndef _ONEDPL_SERIAL_ALGORITHM_IMPL_H
 #define _ONEDPL_SERIAL_ALGORITHM_IMPL_H
 
-#if (__cplusplus >= 201703L)
+#include "common_config.h"
```

Comment:
I am against of including this header to internal headers. The idea of `common_config.h` is to be included as first file in public headers. Having it included in internal headers might hide the error when we add new public one and forget to include common_config there.

Reply:
I'm not going to deny the general idea of this file.
But I should note that these files are located in the same directory.
And from this point of view, it seemed acceptable to me to use the file.

Otherwise what we should do here? Add the new definition for our CPP version macro?

Reply:
@akukanov what do you think about this limitation?

Reply:
Fixed